src/plugins/LADTable/components/LadTableSet.vue

```diff
@@ -76,7 +79,8 @@ export default {
       ladTableObjects: [],
       ladTelemetryObjects: {},
       viewContext: {},
-      configuration: this.ladTableConfiguration.getConfiguration()
+      configuration: this.ladTableConfiguration.getConfiguration(),
+      subscribedObjects: []
```

Comment:
is this supposed to be an object? looks like it's being used as an object below

Reply:
yes this is. Could you please update this?

Reply:
good catch! I've updated it